deploy/kubernetes/elastic-agent-kustomize/onboarding/default/elastic-agent-managed/base/elastic-agent-managed-daemonset.yaml

```diff
@@ -0,0 +1,167 @@
+# For more information https://www.elastic.co/guide/en/fleet/current/running-on-kubernetes-managed-by-fleet.html
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: elastic-agent
+  namespace: kube-system
```

Comment:
Wonder if all `namespace` fields should be removed from all files apart from the `kustomization.yaml`? The value gets correctly overwritten anyway, but it is not doing anything anymore.

Reply:
I will keep it for now for clarity